date-input.js

```diff
@@ -193,7 +195,6 @@ Polymer({
   _computeDate: function(month, year) {
     // Months are 0-11.
     this.date = {month: month, year: year};
-    this.fire('dateChanged', this.date);
```

Comment:
Do you think anyone is relying on that input? Do we have a workaround if they are (i.e listening to `date-changed` instead)? If so: would you mind adding a 'Changes in 3.0' section and documenting this in the readme? 🙏

Reply:
I'll add a section